src/js/view/view.js

```diff
@@ -501,7 +501,17 @@ define([
             _this.trigger(evt.type, evt);
         }
 
+        var toggleControls = function() {
+            if (_model.get('controls')) {
+                _controlsLayer.style.display = '';
+            } else {
+                _controlsLayer.style.display = 'none';
+            }
```

Comment:
It is done. The rule is in "styles.less" but will be moved into view.less after Kyles related change.